cypress/integration/persianBodySpec.js

```diff
@@ -11,7 +11,7 @@ import {
   visibleImageWithCaption,
 } from '../support/bodyTestHelper';
 
-describeForLocalOnly('Article Body Tests', () => {
+describeForLocalAndTest('Article Body Tests', () => {
   // eslint-disable-next-line no-undef
   before(() => {
     cy.visit(`/persian/articles/${config.assets.persian}`);
```

Comment:
To check these would work, I replaced this locally with the absolute test URL
```   cy.visit(`https://www.test.bbc.com/persian/articles/${config.assets.persian}`);```
and the tests continued to pass.